deblur/workflow.py

```diff
@@ -41,9 +47,95 @@ def dereplicate_seqs(seqs_fp,
         log_name=log_name)
 
 
-def remove_artifacts_seqs(seqs_fp):
-    """Step 3: remove artifacts from FASTA file"""
-    pass
+def remove_artifacts_seqs(seqs_fp,
+                          ref_fp,
+                          output_fp,
+                          ref_db_fp=None,
+                          negate=False,
+                          threads=1):
+    """Step 3: remove artifacts from FASTA file using
```

Comment:
I'm not exactly sure what you mean by 'agnostic' in this context, also, you're suggesting to remove the two words "Step 3"  of the description? The functions trim_seqs/dereplicate_seqs (merged) also have the "Step 1"/"Step 2a" preceding their descriptions, remove those as well?